syntax = "proto3";

message FileRequest {
    string path = 1;
    int64 offset = 2;
    int64 size = 3;
}

/*
    FileRequest {
        path = "dupa"
    }
*/